sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -28,6 +28,86 @@ public class TableClient
         private readonly string _version;
         private readonly bool _isPremiumEndpoint;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TableClient"/>.
+        /// </summary>
+        /// <param name="tableName">The name of the table with which this client instance will interact.</param>
+        /// <param name="endpoint">
+        /// A <see cref="Uri"/> referencing the table service.
+        /// This is likely to be similar to "https://{account_name}.table.core.windows.net/" or "https://{account_name}.table.cosmos.azure.com/".
+        /// </param>
+        /// <param name="options">
+        /// Optional client options that define the transport pipeline policies for authentication, retries, etc., that are applied to every request.
+        /// </param>
+        public TableClient(string tableName, Uri endpoint, TableClientOptions options = null)
```

Comment:
Just wondering what the thinking was behind a ctor that doesn't take in `TableClientOptions` versus taking in one that defaults!

Reply:
It's specified in the [guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-client-ctor)